LuaJIT.cmake

```diff
@@ -386,6 +395,8 @@ else()
   make_directory(${CMAKE_CURRENT_BINARY_DIR}/minilua)
   if (HOST_WINE)
     set(MINILUA_PATH ${CMAKE_CURRENT_BINARY_DIR}/minilua/minilua.exe)
+  elseif(ARM64_MSVC)
+    set(MINILUA_PATH ${CMAKE_CURRENT_BINARY_DIR}/minilua/Debug/minilua)
```

Comment:
DEBUG can be replace ${CMAKE_BUILD_TYPE}?

Reply:
I found another way to handle this PR more cleanly. I added another commit which could be more comprehensive how it works. Would you revisit to take a look on it?

Reply:
Hi, is there anything else to be need to handle in this PR?
It's ready to go for x86_64 to arm64 Windows and arm64 Windows but x64 compiler of MSVC.